### ISSUE TITLE
Add support for `/title`

### ISSUE THREAD

**Issue by zardoy** [zardoy/space-squid#5]
dear "easy-money" bounty hunters, if you are real human beings please check context and info more carefully. this issue had already around 10 prs (for the most useless feature in the project btw).

There are other aspects to improve in the project such as more survival features, which you are free to propose for a bounty.

**Comment by IssueHuntBot** [comment]
[@zardoy](https://oss.issuehunt.io/u/zardoy) has funded $10.00 to this issue.

---
- Submit pull request via [IssueHunt](https://oss.issuehunt.io/repos/679440896/issues/5) to receive this reward.
- Want to contribute? Chip in to this issue via [IssueHunt](https://oss.issuehunt.io/repos/679440896/issues/5).
- Checkout the [IssueHunt Issue Explorer](https://oss.issuehunt.io/issues) to see more funded issues.
- Need help from developers? [Add your repository](https://oss.issuehunt.io/r/new) on IssueHunt to raise funds.

**Comment by ArikSquad** [comment]
I could help with this, but is this already done? It seems like some functionality has been done?

**Comment by zardoy** [comment]
Nah. Why do you think so?

**Comment by ArikSquad** [comment]
Which part are you answering to

**Comment by zardoy** [comment]
> It seems like some functionality has been done?

Why do you think it was done? Afaik there is no code related to it. The final goal is in the description we need a new command implementation /title and test it with the old and new syntax (commands against vanilla).

**Comment by ArikSquad** [comment]
I was just making sure, I did see it sending some packets, so I took that as functionality. Even though it was missing some vital ones. Thanks for the clearing and I'll probably look into it later

**Comment by ashishk15678** [comment]
is the issue still open ? looking at #6 , it seems the work is done.

**Comment by zardoy** [comment]
i think you can finish it @ashishk15678 is there anything required from my side?

**Comment by zardoy** [comment]
definitely not enough PRs

**Comment by ProtonsAndElectrons** [comment]
I opened a focused follow-up PR at #20 for the remaining `/title` behavior gap I found on current `everything`: sending `title`/`subtitle` text should not reset previously configured `/title ... times`, while `clear` and `reset` should reject trailing args. Targeted title tests and `pnpm build` pass.

**Comment by ProtonsAndElectrons** [comment]
Thanks for merging #20. I see #5 is still open on GitHub/IssueHunt, and IssueHunt currently has only #16 attached. Does #20 satisfy the remaining `/title` scope for the IssueHunt reward, or is there another gap you'd like me to cover? Happy to follow up either way.

**Comment by ProtonsAndElectrons** [comment]
Hi @zardoy, PR #20 was merged for this IssueHunt-funded issue, but the IssueHunt page currently appears to list a different submitted PR (#16) and not #20.

Is there a preferred way you want me to associate the merged PR with the IssueHunt bounty/award?

**Comment by ProtonsAndElectrons** [comment]
Follow-up on the IssueHunt-funded `/title` issue: I rechecked current `everything` after #20 merged and validated the focused title coverage plus the package build again:

```bash
pnpm vitest src/lib/modules/title.test.ts --run
pnpm build
```

Both pass on the merged head. The IssueHunt page still appears to list only #16 under submitted pull requests, not merged #20. If #20 covers the remaining `/title` behavior, could you let me know the preferred way to associate it with the IssueHunt bounty? If there is still a specific vanilla `/title` gap left, please point me at it and I can follow up with another focused patch.

**Comment by IngARodriguez** [comment]
/attempt #5

I found a remaining legacy-compatibility gap on current everything: pre-1.11 clients use different title packet action IDs, and 1.8-1.10 action bars should go through the chat packet instead of title action 2. I have a focused patch with boundary tests for 1.10 vs 1.11 and I’m opening the PR next.

**Comment by IngARodriguez** [comment]
PR submitted: https://github.com/zardoy/space-squid/pull/24

**Comment by lkaka85** [comment]
I can implement the `/title` command by analyzing the existing command structure and integrating it with consistent input handling and output formatting. Expect delivery within 24 hours.
